guide/03-the-gis/administering-your-gis.ipynb

```diff
@@ -1 +1,2405 @@
\ No newline at end of file
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Administering your GIS\n",
+    "\n",
+    "ArcGIS administrators can leverage the `gis.admin` module of **ArcGIS API for Python** to assist with and automate their administrative tasks. These tasks can include anything from checking the status of servers, assigning licenses to named user accounts, or modifying the GIS's look and feel.\n",
+    "\n",
+    "ArcGIS Online Organizations (AGOL) and ArcGIS Enterprise instances vary in the amount of customization you can make. Enterprise organizations can be customized much more than ArcGIS Online organizations as Enterprise allows administrators full access. No matter what your organization is, the API and usage are identical. The table below illustrates the extent to which each can be customized and administered."
```

Comment:
Does `Organizations` need to be capitalized?  

Change:
`No matter what your organization is`
to:
`no matter your organization`